tests/test_pii_docs/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "test_pii_docs"
+version = "0.1.0"
+edition = "2021"
+
+
+[workspace]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
```suggestion
```

Do we also need the `workspace` attribute?

Reply:
It's pretty much a dummy file, the crate is never actually run, just for testing purposes. There just need to be a `Cargo.toml` file existing, and i created with normal `cargo new` command.